madhumitha/images.html

```diff
@@ -0,0 +1,32 @@
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title>image</title>
+</head>
+<body>
+    <div>
+        <img id="image_id"></img>
+    </div>
+    <div>
+        <button id="button">Show Image</button>
+    </div>
+    <script>
+        document.addEventListener("DOMContentLoaded", function () {
+            let x = document.getElementById("button");

```

Comment:
Avoid naming variables like 'x'.

Reply:
okay, i have done the changes and pushed the code